### PR TITLE
Added reference to http.server

### LINKED ISSUE #233: Contributing page uses SimpleHTTPServer
This has been merged into http.server for Python 3 (as seen in the [SimpleHTTPServer](https://docs.python.org/2/library/simplehttpserver.html) docs).

References to this for Python 3 should be changed to reflect this.

### PR DESCRIPTION
**Pull request by jonpwilson**
Fixes #233 

### This pull request's proposal

Adds a 'http.server' command when starting a Python 3 server
Removed a minor typo ('extensionS' for a singular)

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by phillxnet** [comment]
@jonpwilson Hello again, and this is a nice find / improvement.

But on my build in Sphinx of master with this pull request applied I get the following warning:

```
contribute_documentation.rst:144:Literal block expected; none found.
```
So it looks like at least here that Sphinx is not happy with this change.

Our semi automated doc pr tester is set to halt on warnings to help us avoid a build of issues.

Let me know if you fancy addressing this or I can take a look and make a suggestion.

Running Sphinx v1.7.6

File: contribute_documentation.rst [hunk 1]
```diff
@@ -139,10 +139,16 @@ following command ::
 
 HTML files are generated in _build/html directory. From a separate terminal
 window, you can have a simple Python webserver always serving up this content
-with the following command ::
+with one of the following commands ::
```
Comment (phillxnet) [inline contribute_documentation.rst]:
```suggestion
with one of the following commands:
```

Comment (phillxnet) [reply contribute_documentation.rst]: @jonpwilson So had another look and it seems at least my version of Sphinx is worried about this bit.
By leaving this prior literal block indicator in place it throws the "Literal block expected; none found." error so I'm suggesting we pop in a straight colon here.

**Comment by jonpwilson** [comment]
Hi @phillxnet,

Interesting - I knew that I'd run it before (I read the contributing guide which is why I spotted this!) so tried again and it works for me - using Sphinx v2.4.1 & Python 3.7.5.

What versions are you running? Just asking as I was looking at the Circle-CI / Github actions ticket and it'd be good to get your version in there to prevent this sort of thing happening again.

Thanks

**Comment by phillxnet** [comment]
@jonpwilson Hello again. Apologies for the delay getting back to this one.

Re:

> Interesting - I knew that I'd run it before (I read the contributing guide which is why I spotted this!) so tried again and it works for me - using Sphinx v2.4.1 & Python 3.7.5.

I know, but it was only a warning but we have our build system to halt on warnings. My version, in the build system, is:

> Running Sphinx v1.7.6

and I got the same warning with v1.6.5, these are the distro default versions for the systems in question. They may have taken this out as a warning in the interim. No worries, now you've accepted that change we are good to do and it builds without warning and renders as expected on a local Apache setup. So about to merge and publish.

Thanks for yet another pull request. Much appreciated.

Re:

> I was looking at the Circle-CI / Github actions ticket and it'd be good to get your version in there to prevent this sort of thing happening again.

Yes I think it would be good to get this setup. I think it's more about the Sphinx version really and a Sphinx build check would be a great candidate for a GitHub action. The Circle-CI element might be unnecessarily extending our dependencies for infrastructure and it's yet another account to manage. We will have to circle back around to this though as I think it's going to be one of the easiest things to integrate into actions now that they are out of beta and we have reduced our Sphinx plugin dependencies with the recent YouTube embed removals pr merge.

**Comment by phillxnet** [comment]
@jonpwilson These changes are now published. Thanks again and nice improvement.

**Pull request merged**